sktime/forecasting/model_selection/_tune.py

```diff
@@ -302,6 +302,16 @@ def evaluate_candidates(candidate_params):
         self._is_fitted = True
         return self
 
+    @property
+    def cutoff(self):
+        """The time point at which to make forecasts
+
+        Returns
+        -------
+        cutoff : pd.Period, pd.Timestamp, int
+        """
+        return self.best_forecaster_.cutoff
```

Comment:
what happens if the algorithm hasn't been fitted yet?

Reply:
it would throw an error that there is no `best_forecaster_` i think, is this ok like that?

Reply:
probably best to check `self.check_is_fitted()`